common/app/model/content.scala

```diff
@@ -197,15 +197,17 @@ final case class Content(
     } getOrElse (None,None)
 
     val articleMeta = if (tags.isUSMinuteSeries) {
-      Some("isMinuteArticle", JsBoolean(tags.isUSMinuteSeries))
-    } else None
+      List(
+        Some("isMinuteArticle", JsBoolean(tags.isUSMinuteSeries)),
+        Some("enhanceMobileTweets", JsBoolean(true))
```

Comment:
It's always true if it is the USMinuteSeries as inside the if statement. Seemed redundant to add tags.isUSMinuteSeries here too but probably doing it wrong :/